cli/definitions/cli.ts

```diff
@@ -113,7 +113,6 @@ export interface ILoginCommand extends ICommand {
 }
 
 export interface ILogoutCommand extends ICommand {
-    isLocal: boolean;
 }
```

Comment:
I feel like this breaks consistency and type annotation, e.g. this interface is referred to in lines 457 and 559 of `command-executor.ts`. What do you think?